File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/emr/OCS2015U31EmrTagGetTagInfo.java
package nta.med.data.model.ihis.emr;


public class OCS2015U31EmrTagGetTagInfo {
	private Integer tagId;
	private String tagCode;
	private String tagName;
	private String tagDisplayText;

	public OCS2015U31EmrTagGetTagInfo(Integer tagId, String tagCode,
			String tagName, String tagDisplayText) {
		super();
		this.tagId = tagId;
		this.tagCode = tagCode;
		this.tagName = tagName;
		this.tagDisplayText = tagDisplayText;
	}

	public Integer getTagId() {
		return tagId;
	}

	public void setTagId(Integer tagId) {
		this.tagId = tagId;
	}

	public String getTagCode() {
		return tagCode;
	}

	public void setTagCode(String tagCode) {
		this.tagCode = tagCode;
	}

	public String getTagName() {
		return tagName;
	}

	public void setTagName(String tagName) {
		this.tagName = tagName;
	}

	public String getTagDisplayText() {
		return tagDisplayText;
	}

	public void setTagDisplayText(String tagDisplayText) {
		this.tagDisplayText = tagDisplayText;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/OcsEmr/OCS2015U31EmrTagGetTagInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.OcsEmr
{
    [Serializable]
    public class OCS2015U31EmrTagGetTagInfo
    {
        private String _tagId;
        private String _tagCode;
        private String _tagName;
        private String _tagDisplayText;

        public String TagId
        {
            get { return this._tagId; }
            set { this._tagId = value; }
        }

        public String TagCode
        {
            get { return this._tagCode; }
            set { this._tagCode = value; }
        }

        public String TagName
        {
            get { return this._tagName; }
            set { this._tagName = value; }
        }

        public String TagDisplayText
        {
            get { return this._tagDisplayText; }
            set { this._tagDisplayText = value; }
        }

        public OCS2015U31EmrTagGetTagInfo() { }

        public OCS2015U31EmrTagGetTagInfo(String tagId, String tagCode, String tagName, String tagDisplayText)
        {
            this._tagId = tagId;
            this._tagCode = tagCode;
            this._tagName = tagName;
            this._tagDisplayText = tagDisplayText;
        }

    }
}